wiki/BanchoBot/en.md

```diff
@@ -2,36 +2,48 @@
+| `/help` | N/A | View this help. | /help | The list of available commands is given. |
+| `/ignore` | `<user>[@chp]` *(By adding an @ followed by the letters c, h, and/or p, you may ignore them in chat, highlights, or PMs respectively.)* | Ignore all messages from specified user for this session. | /ignore ShibaInu @c | "You will no longer hear ShibaInu. {chat}" |
+| `/unignore` | `<user>` | Stop ignoring this user for this session. | /unignore ShibaInu | "You may now hear ShibaInu." |
+| `/keys` | N/A | Key-specific help. | /keys | "PageUp/PageDown Scroll the chat window. You may also use the mouse wheel. <br> Tab Autocomplete currently typing nickname. <br> F8 Toggle chat. <br> F9 Toggle extended chat. <br> Ctrl-C/V Copy/Paste. <br> Alt+0-9 Switch to respective tab. |
+| `/me` | `<action>` | Perform a third-person action. | /me is playing osu!mania | "\<username\> is playing osu!mania" |
+| `/np` | N/A | Print to chat the song you are listening to or playing. | /np | "\<username\> is listening to [Soleily - Renatus](https://osu.ppy.sh/b/557820) |
+| `/reply` or `/r` | `<message`> | Reply to the last received private message. | /r I love this beatmap too! | (In a friend's chat tab) "TomAto: I love this beatmap too!" *Note that you will not automatically switch tabs.* | 
+| `/savelog` | N/A | Save recent chat to a text file. | /savelog | (In a text bubble in the bottom right) "Exported chat log to Chat/#osu!-20170530-053420.txt." *The first jumble of numbers is the date, in the format of YYYYMMDD. The second is the time, in the format of HHMMSS. The time does not discern between AM and PM.* |
+| `/watch` | `<user>` | Start spectating `<user>`. | /watch Firestar | (In a text bubble in the bottom right) "Started watching Firestar". *Note: You need to have the beatmap the person you're spectating is playing with. If they are not currently playing a beatmap, then you will remain where you were until they do enter one. To stop spectating, hit ESC once they start playing, similar to ending a replay. Also bear in mind that the person you are spectating will be notified that you are spectating them.* |
+| `/nopm` | N/A | Quickly toggle whether you want to receive private messages from friends or not. | /nopm | (Displayed as a banner in the center of the screen) "You are now ignoring all private messages from everyone except your friends." When toggled again, "You are not longer ignoring pivate messages from non-friends." |
+| `/invite` | `<user>` |  Sends a multiplayer invitation to the specified user with a link to the room *(You must be in a match to invite someone.)* | /invite PikaFan025 | (Displayed as a banner in the center of the screen) "PikaFan025 has been invited to your game." On the recipient's end, a chat tab with you opens (if it hasn't already) and the text "\<username\>: Come join my multiplayer match: "[\<Room name\>](/wiki/BanchoBot/en.md)" |
+
+### `/bb` Commands
```

Comment:
Note: I couldn't figure out how `SEARCH` worked, so I omitted it. If anyone else knows, feel free to comment.